gdbinit.py

```diff
@@ -19,39 +19,50 @@
 virtual_env = environ.get("VIRTUAL_ENV")
 
 if virtual_env:
-    no_venv_warning = environ.get("PWNDBG_NO_VENV_WARNING", 0)
-    if not no_venv_warning and not sys.executable.startswith(virtual_env):
```

Comment:
Before this PR/commit, if PWNDBG_NO_VENV_WARNING=1 was set, then we never set the virtualenv packages because `sys.path.append(virtualenv_site_packages)` was in that if branch.

Meaning, it was not possible to suppress the warning...